packages/ocp_reveal/ocp_reveal.0.1/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "OCaml bindings for Reveal.js, an HTML presentation framework"
+description: "OCaml bindings for Reveal.js, an HTML presentation framework"
+maintainer: ["OCamlPro <contact@ocamlpro.com>"]
+authors: ["Çagdas Bozman"]
+license: "LGPL"
+homepage: "https://github.com/OCamlPro/ocp-reveal"
+doc: "github.com/OCamlPro/ocp-reveal"
+bug-reports: "https://github.com/OCamlPro/ocp-reveal/issues"
+depends: [
+  "ocaml" {>= "4.05"}
```

Comment:
```suggestion
  "ocaml" {>= "4.05" & < "4.11"}
```
The failure seems to indicate incompatibility with ocaml 4.11

Reply:
Because in that case I would merge directly, without changes

Reply:
I don't see why it wouldn't work with 4.11.